docs/refactoring/sonarcloud-report.md

```
@@ -0,0 +1,24 @@
+# SonarCloud Результати
+
+## До: 
+![](sonar-before1.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)